src/main/java/com/cloudbees/jenkins/plugins/kubernetes_credentials_provider/SecretUtils.java

```diff
@@ -108,10 +108,12 @@ public static String base64DecodeToString(String s) {
      * @return the scope for a given secret.
      */
     public static CredentialsScope getCredentialScope(Secret s) {
-        return CredentialsScope.valueOf(Optional
+        CredentialsScope scope = CredentialsScope.valueOf(Optional
                 .ofNullable(s.getMetadata().getLabels().get(JENKINS_IO_CREDENTIALS_SCOPE_LABEL))
                 .orElse("global")
                 .toUpperCase(Locale.ROOT));
+        LOG.log(Level.FINE, "Scope for secret {0} is set to {1}", new Object[] {getCredentialId(s), scope});
```

Comment:
this would only be useful in the happy case.  in the case `scope` was set to "barf"  this would throw an `IllegalArgumentException` as `CredentialsScope.valueOf("barf")` is invalid.
"user" is a valid option for the `CredentialScope` but not something that this code supports.

Reply:
Ah, I must have misunderstood what you meant. So, only if it throws an exception, then I log to fine with the secret id. Will push a fix asap.

Reply:
Discovered that the unit test I wrote for this case with an unsupported scope label had a bug where I had managed to test the wrong label. Then I got the `IllegalArgumentException` you mentioned. Thanks for good feedback so far! :+1:

Reply:
Does it look better now, @jtnord ?